openlibrary/i18n/es/messages.po

```diff
@@ -4015,7 +4015,7 @@ msgstr "Disponibilidad"
 
 #: type/work/editions_datatable.html:37
 msgid "Add another edition"
```

Comment:
```suggestion
msgid "Add another edition?"
```
Question mark is needed here in order to match the new `msgid` in the template and front-end code.